components/availability/src/Availability.svelte

```diff
@@ -221,6 +227,20 @@
           availability = AvailabilityStatus.BUSY;
         }
 
+        if (
+          availability === AvailabilityStatus.PARTIAL &&
+          required_participants.length &&
+          availability === AvailabilityStatus.PARTIAL
```

Comment:
I think there's a repeated condition in here
```suggestion
          required_participants.length &&
          availability === AvailabilityStatus.PARTIAL
```

Reply:
Good catch!